notes/appeals.vbs

```diff
@@ -90,6 +92,8 @@ Do
 		Dialog Dialog1
 		cancel_without_confirmation
       	Call validate_MAXIS_case_number(err_msg, "*")
+        IF docket_number = "" then err_msg = err_msg & vbNewLine & "* Please enter a docket number or enter N/A if unknown."
+        IF appeal_actions <> "Received" AND appeal_actions <> "Summary Completed" AND isnumeric(docket_number) = False then err_msg = err_msg & vbNewLine & "* You have selected an action that should correspond to an existing docket number. Please enter a numeric docket number."
```

Comment:
Line 96 - I think this can be removed now since docket number is now universal.

Reply:
This is needed to allow for script actions that do not require docket number.